pkg/reconciler/testing/auditlogs.go

```diff
@@ -200,8 +199,8 @@ func WithCloudAuditLogsSourceAnnotations(Annotations map[string]string) CloudAud
 	}
 }
 
-func WithCloudAuditLogsSourceDefaultGCPAuth() CloudAuditLogsSourceOption {
+func WithCloudAuditLogsSourceSetDefaults() CloudAuditLogsSourceOption {
```

Comment:
if you are not passing arguments to the method I think you can just do...

`func WithCloudAuditLogsSourceSetDefaults(s *v1beta1.CloudAuditLogsSource) {
	s.SetDefaults(gcpauthtesthelper.ContextWithDefaults())
}`

and the caller code will just be `WithCloudAuditLogsSourceSetDefaults` instead of `WithCloudAuditLogsSourceSetDefaults()` everywhere...